src/services/storage.cpp

```diff
@@ -235,6 +235,18 @@ bool Storage::putBlockBody(json &data, const string &block_id) {
     }
   }
 
+  string duplicate_node;
+  size_t mtree_size = data["mtree"].size();
+  for (size_t i = 1; i < mtree_size; ++i) {
```

Comment:
```suggestion
  for (size_t i = mtree_size - 1; i > 0; --i) {
    if(data["mtree"][i].get<string>() == data["mtree"][i-1].get<string>())
      data["mtree"][i] = "";
  }
```
238~248번째 라인까지 이렇게 하면 됩니다. 참 쉽죠?^^

Reply:
이런 것은 앞에서부터 하는 것보다 뒤에서부터 확인하고 고치면 편합니다.

Reply:
아아 그렇네요 변경하여 반영하겠습니다